jupyterlab/environment.yaml

```
@@ -58,6 +58,7 @@ dependencies:
   - pip:
       # vscode jupyterlab launcher
       - git+https://github.com/betatim/vscode-binder
+      - jupyterlab_nvdashboard==0.10.0
```

this package is on conda-forge but the recipe appears unmaintained, the PR to bump to 0.10.0 is hanging here: https://github.com/conda-forge/jupyterlab-nvdashboard-feedstock/pull/17

---

Yeah, we can reach out to their team about that.

I think we can still install it through PyPI, which seems to be the latest version: https://pypi.org/project/jupyterlab-nvdashboard/

---

@pavithraes it looks like a new release fixing the issue that was blocking us here is out, can we change the pin to [v0.11.0](https://github.com/rapidsai/jupyterlab-nvdashboard/releases/tag/v0.11.0) here?

```suggestion
      - jupyterlab_nvdashboard==0.11.0
```

---

Is this something we want to incorporate in the next release of Nebari? Just trying to figure out what pending PRs we have to cut a final release this week.

---

Thanks! This should now be ready to merge :)